docs/plugins/python.md

```diff
@@ -73,7 +73,7 @@ This plugin also enforces a set of allowed Python versions.  If you choose a
 Python version that is not allowed, you will see an error messages such as:
 
 ```
-> Python 3.2 not allowed; choose from [2.6, 2.7, 3.4, 3.5, 3.6]
+> Python 3.2 not allowed; choose from [2.7, 3.4, 3.5, 3.6]
```

Comment:
2.7, 3.5, 3.6, 3.7 perhaps?

Reply:
Yes, let's officially drop 2.6 & 3.4 and add 3.7.